src/hiopbbpy/opt/boalgorithm.py

```diff
@@ -190,16 +234,19 @@ def optimize(self):
     self.y_hist = []
 
     for i in range(self.bo_maxiter):
-      print(f"*****************************")
-      print(f"Iteration {i+1}/{self.bo_maxiter}")
+      self.logger.critical(f"*****************************")
+      self.logger.critical(f"Iteration {i+1}/{self.bo_maxiter}")
+
+      prev_best = np.min(y_train)
 
       y_train_virtual = y_train.copy() # old training + batch_size num of virtual points
       for j in range(self.batch_size):
         # Get a new sample point
+        self.logger.scalars(f"In batch {j+1}/{self.batch_size}")
```

Comment:
This is not the batch number but rather the `j`th point of a given batch. `In batch {j+1}/{self.batch_size}` is misleading.